MonoGame.Framework/Input/JoystickCapabilities.cs

```diff
@@ -22,6 +22,12 @@ public struct JoystickCapabilities
         public string Identifier { get; internal set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether the joystick is a gamepad.
```

Comment:
The docs here are wrong.  The user cannot "set" this property... it is internal.